### PR TITLE
set notebook padding at label level not tab, fixing resizing error

### PR DESCRIPTION
**Pull request by dterrahe**
#6375 hardcoded the padding in notebook tabs because many attempts to find a way to pick up the actual padding set in css failed.

Then https://github.com/darktable-org/darktable/commit/697ce8790311c6659f5700c3813985ec28349e46 made padding dependent on font size, which, for large fonts or high resolution monitors, caused the resizing code to generate
`gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkNotebook
`

After a few more hours of trying to get the correct GtkStyleContext for the tab, I tried if setting the padding at the tab _label_ level made a significant difference visually. I didn't see any.

And now it is no longer necessary to adjust the allocatable width for the padding at all, since it now falls _within_ the tabs and is dealt with by ellipsizing.

@Nilvus your comments please. I haven't merged the two successive section of `notebook tab label` since the second has the comment "_disabled tabs_", although I don't understand why...

### CODE REVIEW

**Review by TurboGit — APPROVED** [review]
Works for me. I'll let @Nilvus review before merging.

**Review by Nilvus — CHANGES_REQUESTED** [review]
Works great on my side. A better implementation of that. Thanks @dterrahe

> @Nilvus your comments please. I haven't merged the two successive section of `notebook tab label` since the second has the comment "_disabled tabs_", although I don't understand why...

About that, it's better to merge them now, that will save lines. You could also remove "disabled" comments. This part, just set notebook tabs in inactive state and on another part of this CSS, active/hover states are then set.

**Comment by dterrahe** [comment]
> it's better to merge them now

done

**Review by Nilvus — APPROVED** [review]

**Review by TurboGit — APPROVED** [review]
Thanks!

**Pull request merged**
